actions/java-gradle-publish/action.yaml

```diff
@@ -61,7 +61,7 @@ runs:
         gradle-cache-read-only: ${{ inputs.gradle-cache-read-only }}
 
     - name: Publish snapshots to Nexus
-      if: ${{ !startsWith(github.ref, 'refs/tags/') }}
+      if: ${{ !startsWith(github.ref, 'refs/tags/') && !contains(github.repository, 'playground') }}
```

Comment:
Looks good, can you also do this for the assessment an publication of code quality to sonaqube?
https://github.com/bakdata/ci-templates/blob/main/actions/java-gradle-assess-code-quality/action.yaml#L72
https://github.com/bakdata/ci-templates/blob/main/actions/java-gradle-assess-code-quality/action.yaml#L80